contracts/crafting/libraries/LibCrafting.sol

```diff
@@ -10,28 +10,44 @@ pragma solidity ^0.8.0;
 
 import {IDiamondCut} from "../../diamond/interfaces/IDiamondCut.sol";
 
+struct CraftingInputItem {
+    uint256 tokenType; // item type: 20 for ERC20, 1155 for ERC1155
+    address tokenAddress; // address of the token
+    uint256 tokenId; // id of the token, any number for erc20
+    uint256 amount; // amount of the token
+    uint256 tokenAction; // 0 for transfer, 1 for burn, 2 for hold
+}
+struct CraftingOutputItem {
+    uint256 tokenType; // item type: 20 for ERC20, 1155 for ERC1155
+    address tokenAddress; // address of the token
+    uint256 tokenId; // id of the token, any number for erc20
+    uint256 amount; // amount of the token
+    uint256 tokenAction; // 0 mint, 1 transfer
```

Comment:
Let us treat 0 as a default value. Should the default output method be `mint` or `transfer`?

Reply:
This comment is inconsisntent with:

```
uint256 constant OUTPUT_TOKEN_ACTION_TRANSFER = 0;
uint256 constant OUTPUT_TOKEN_ACTION_MINT = 1;
```